sandbox/grist/docmodel.py

```diff
@@ -58,6 +59,22 @@ def tzinfo(rec, table):
       except KeyError:
         return moment.TZ_UTC
 
+    def parsed_settings(rec, table):
+      try:
+        return json.loads(rec.documentSettings)
+      except ValueError:
+        return {}
+
+    def locale(rec, table):
+      return rec.parsed_settings.get("locale", "en-US")
```

Comment:
I remember seeing this default value somewhere else. Should we have it as a global variable ?

Reply:
Makes sense, I pushed it to a global constant.